src/MatrixClientPeg.js

```diff
@@ -71,7 +72,21 @@ class MatrixClientPeg {
         const opts = utils.deepCopy(this.opts);
         // the react sdk doesn't work without this, so don't allow
         opts.pendingEventOrdering = "detached";
-        this.get().startClient(opts);
+
+        let promise = q();
+        if (this.matrixClient.store instanceof Matrix.IndexedDBStore) {
```

Comment:
Ew. Again, if we're making things OO, can we make them good OO and avoid instance of (implement a startup() method that nops on the other stores).